### ISSUE TITLE
code bug in ChannelPool

### ISSUE THREAD

**Issue by leedewdew** [Jongchan/attention-module#30]
`class ChannelPool(nn.Module):
    def forward(self, x):
        return torch.cat( (torch.max(x,1)[0].unsqueeze(1), torch.mean(x,1).unsqueeze(1)), dim=1 )
`
torch.max(x,1)[0] is very strange here. 
Is that a bug?
Why we need get the feature map of the first image?
@Jongchan

**Comment by Tsatthienmach** [comment]
I'm confused with this just like u, so i found in torch documents, and it demonstrates that when using torch.max(x ,dim) returns [max matrix, indict matrix] , [0] helps getting a max matrix. :+1:

**Comment by leedewdew** [comment]
> I'm confused with this just like u, so i found in torch documents, and it demonstrates that when using torch.max(x ,dim) returns [max matrix, indict matrix] , [0] helps getting a max matrix. 👍

You are perfectly right. Thanks very much!
